tests/queries/0_stateless/03270_object_to_json_alter_wide_merge_tree.sql

```diff
@@ -1,5 +1,5 @@
--- Tags: long, no-azure-blob-storage
--- no-azure-blob-storage: too slow
+-- Tags: long, no-s3-storage, no-azure-blob-storage
+-- no-s3-storage, no-azure-blob-storage: too slow
```

Comment:
I would say that almost always explicit (either S3 or Azure) exclude is wrong (especially explicit S3, since Azure is slower), so better to use `no-object-storage`

```suggestion
-- Tags: long, no-object-storage
-- - no-object-storage: too slow
```

Though it has the same meaning, looks simpler (and here https://github.com/ClickHouse/ClickHouse/pull/77344 I have unified existing)